### ISSUE TITLE
Enable inferencing for various model frameworks/outputs

### ISSUE THREAD

**Issue by nrweir** [CosmiQ/solaris#23]
@nrweir commented on [Wed Apr 17 2019](https://github.com/CosmiQ/cw-nets/issues/56)

Will need to look into what the outputs are like for various different models and see how I should store them, as well as how inference is run for:

- [ ] PyTorch
- [ ] tf/keras
- [ ] Darknet

Ideally I'd like to be able to enable sliding windows to cover images for things like segmentation masks.
